recipes/delft_fiat/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set name = "delft_fiat" %}
+{% set version = "0.1.0rc1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 5be537ec4d21cd3d452050e11aa40ed35fc2343bd9d61609a21858adaea42749
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - fiat = fiat.cli.main:main
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python >=3.9
+    - pip
+  run:
+    - gdal >=3.5
+    - numpy
+    - python >=3.9
+    - regex
+    - tomli
+
+test:
+  imports:
+    - fiat
+  requires:
+    - pip
+  commands:
+    - pip check
+
+about:
+  home: https://github.com/simplejson/simplejson
```

Copy-n-pasta. Please use [grayskull](https://github.com/conda/grayskull) to generate the recipes to help with that.
```suggestion
  home: https://github.com/Deltares/Delft-FIAT
```